### DISCUSSION TITLE
Tablet with pressure detection glitching out

### DISCUSSION THREAD

**Discussion by yesseruser** [ppy/osu#24778] [Q&A] [answered]
### Type

Game behaviour

### Bug description

When playing with a tablet that supports pressure detection (One by Wacom - CTL-472) that and pressing firmly the cursor just glitches out. (See video)
This tablet works normally in stable using system drivers and it would be great if the same worked on Lazer.

### Screenshots or videos

https://1drv.ms/v/s!Ag98u2Nn2vf2mv1yJrm0JdsJ8GOPmA?e=6Pj5EN
(Github didn't let me upload the video because it's too big so I uploaded it to OneDrive)

### Version

2023.908.2-lazer

### Logs

[performance.log](https://github.com/ppy/osu/files/12568638/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12568639/runtime.log)
[updater.log](https://github.com/ppy/osu/files/12568640/updater.log)
[database.log](https://github.com/ppy/osu/files/12568641/database.log)
[input.log](https://github.com/ppy/osu/files/12568642/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/12568643/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/12568644/network.log)

**Comment by ItsShamed** [comment]
Maybe try disabling in-game tablet support and raw input

**Reply by yesseruser** [reply]
Enabling raw input seems to work.

**Reply by yesseruser** [reply]
But it does give the game lagspikes and clicking on the tablet offsets the cursor to the left which is annoying.

**Reply by yesseruser** [reply]
The offset makes it unusable when navigating basically any menu making me use the mouse.

**Reply by yesseruser** [reply]
I changed it because the tablet is actually supported.

**Comment by yesseruser** [comment, marked as answer]
Installing OpenTabletDriver instead of the tablet-provided driver has fixed the issue.
